mavproxy/source/docs/getting_started/download_and_installation.rst

```diff
@@ -20,58 +20,49 @@ http://firmware.ardupilot.org/Tools/MAVProxy/.
     Windows package of MAVProxy. This is being investigated and will
     hopefully be fixed soon. See https://github.com/ArduPilot/MAVProxy/issues/129 for more details.
 
+Alternatively, Windows Subsystem for Linux (WSL) can be used to run MAVProxy from a Linux environment
+within Windows. It generally provides better stability than the native Windows installer above.
+
+
 .. _mavproxy-downloadinstalllinux:
 
 Linux
 =====
 
-First, a few pre-requisite packages need to be installed.
+MAVProxy can be installed via package managers. Select the appropriate set of commands for your Linux variant below.
 
-For Debian based systems:
+For Debian based systems (including WSL):
 
 .. code:: bash
 
     sudo apt-get install python3-dev python3-opencv python3-wxgtk3.0 python3-pip python3-matplotlib python3-pygame python3-lxml python3-yaml

```

Comment:
Unfortunately, this list is not true on recent ubuntu system
Wxgtk4.0 should be used on ubuntu 20.04

Reply:
Added note